tutorials/ab3AddRemark.md

```diff
@@ -52,6 +57,25 @@ public class RemarkCommand extends Command {
 
 Now that we have our `RemarkCommand` ready to be executed, we need to update `AddressBookParser#parseCommand()` to recognize the `remark` keyword. Add the new command to the `switch` block by creating a new `case` that returns a new instance of `RemarkCommand`.
 
+<panel header="Create `RemarkCommandTest`">
+
+- Since we created a new file RemarkCommand, we should also create a `RemarkCommandTest` file. JUnit tests are annotated with `@Test`. For now, we expect `execute` in `RemarkCommand` to fail. Thus, the test will reflect this expectation.
+
+- ```java
+    assertCommandFailure(new RemarkCommand(), model, MESSAGE_NOT_IMPLEMENTED_YET);
```

Comment:
Should we include the full test instead? For clarity and consistency 
```java
@Test
 public void execute() {
     assertCommandFailure(new RemarkCommand(), model, MESSAGE_NOT_IMPLEMENTED_YET);
 }
 ```

Reply:
Okay, done!